resources/com.sap.piper/pipeline/abapEnvironmentPipelineDefaults.yml

```
@@ -47,6 +47,7 @@ stages:
     abapSystemSizeOfPersistence: 2
     abapSystemSizeOfRuntime: 1
     confirmDeletion: 'true'
+    integrationTestOption: 'addOnDeployment'
```

addonDeployment would be incompatible with the current pipeline...
```suggestion
    integrationTestOption: 'systemProvisioning'
```